.github/workflows/build.yml

```diff
@@ -73,4 +73,4 @@ jobs:
         run: sbt flywayMigrate
 
       - name: Build and run tests
-        run: sbt "project server" ++${{matrix.scala}} test doc
+        run: sbt "project server" ++${{matrix.scala}} testIT doc
```

Comment:
Why `testIT` and not `test`?

Reply:
Also,`name: Build and run tests` is very general, please let's be more descriptive

Reply:
Also, this step does not contain unit tests of server, why?

Reply:
Names improved.

Reply:
Fixed in call.